R/html-tweak.R

```diff
@@ -196,7 +196,7 @@ class_page_header <- function(bs_version, header) {
   if (bs_version == 3) {
     return(paste0("<div class='page-header'>", header, "</div>"))
```

Comment:
Since you're in here, this shouldn't be an early return since the two branches are of equal "weight"